From 117812857edecda881562697bdad7dfd2903ecc2 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Mon, 29 Aug 2016 20:15:02 +0000
Subject: [PATCH] The node-fast2 repository on GitHub has been renamed to
 node-fast.  To avoid dealing with rename in Gerrit, I'm going to reimport
 "node-fast" and decommission "node-fast2".  To decomission "node-fast2", I'm
 changing its parent project from GitHub-Replication so that it will be no
 longer replicated.

Change-Id: Ic07eef7f1d65add114bd922a2c6d729131de5dc6
---
 project.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/project.config b/project.config
index 56beaf3..5021f8c 100644
--- a/project.config
+++ b/project.config
@@ -2,7 +2,7 @@
 	description = "Decommissioned.  This was the original Gerrit version of what is now the \"node-fast\" project."
 	state = read only
 [access]
-	inheritFrom = GitHub-Joyent
+	inheritFrom = All-Projects
 [submit]
 	action = fast forward only
 	mergeContent = false
-- 
2.21.0

